src/Crossword.xml

```diff
@@ -9,11 +9,11 @@
          <p>Copyright (C) 1995-2009 Gerd Neugebauer</p>
       </titleText>
 
-      <p>cwpuzzle.dtx is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY. No author or
+      <p><alt match=".+" name="projectName1">cwpuzzle.dtx</alt> is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY. No author or
          distributor accepts responsibility to anyone for the consequences of using it or for whether it serves
          any particular purpose or works at all, unless he says so in writing.
         <br/>.
```

Comment:
Noted this at https://github.com/spdx/license-list-XML/issues/2093#issuecomment-1691452184 and see https://spdx.org/licenses/Crossword.html -- this looks like a stray period that was copied over from the Fedora wiki. 

Can we either remove this, or else mark it as optional? Otherwise I don't think this will match with the new example at https://mirror.las.iastate.edu/tex-archive/macros/latex/contrib/gene/productbox/README.

Reply:
good catch, I think we can remove it to match the original